### PR TITLE
update FE doc

### PR DESCRIPTION
**Pull request by ryestew**

### CODE REVIEW

File: docs/file_explorer.md [hunk 1]
```diff
@@ -1,68 +1,108 @@
...
+- Remixd is a script that you run on your computer that shares a specified folder on your computer with Remix IDE.
+- Remix Desktop is a version of Remix IDE in an Electron app. When using Remix Desktop, you cannot use a browser wallet like MetaMask, because Remix Desktop does not run in a browser.  You'd need to connect using a service like Infura.
+
+### Connecting Remix to remote Git repos
+**If you are not using Remixd, it is highly recommended that you save to a remote repo.** 
+(Browsers do crash causing localstorage & indexedDB to be corrupted)
 
-If you want to use browser storage, but also to save a git repo on IPFS, use the **DGIT** plugin.  
+Remix IDE can be connected to remote Git repos both on services like GitHub or repos in IPFS.  
+Most of the Git operations are done in the **DGIT** pluin. (DGIT stands for Decentralized GIT).
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:35]: I think filip refrain to use the word `Decentralized GIT`

File: docs/file_explorer.md [hunk 2]
```diff
@@ -1,68 +1,108 @@
...
 Also see this article about [securing your files in Remix](https://medium.com/remix-ide/securing-you-file-in-remix-how-to-clone-and-push-f1350111aa13?source=friends_link&sk=a3dbd0d3b0b44a29a28e8c10f8821fde)
 
-File Explorer Tour
--------------------
 
-![](images/a-fe-tour.png)
+Workspaces
+------------
+
+Workspaces in Remix are special folders that separate projects.  Files in one Workspace cannot be accesses from another Workspace.  Workspaces are selected using the Workspace select box.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:46]: `accesses ` => `accessed`

File: docs/file_explorer.md [hunk 3]
```diff
@@ -1,68 +1,108 @@
...
+
+![](images/a-fe-workspaces1.png)
+
+#### New Workspaces
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:50]: It should be `New Workspace`

File: docs/file_explorer.md [hunk 4]
```diff
@@ -1,68 +1,108 @@
...
 
-The book icon - **G.** is the link to the documentation (this page).
+When making a new workspace, Remix offers the following templates: 
 
-### Workspaces
-Workspaces help to organize your files by allowing you to separate your projects.
-Here are the basic operations of managing a Workspace. The letters in bold below refer to the labels in fig. 1.
+- Blank
+- Remix Default
+- Open Zeppelin ERC20
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:59]: Make `Open Zeppelin` a single word

File: docs/file_explorer.md [hunk 5]
```diff
@@ -1,68 +1,108 @@
...
+- Open Zeppelin ERC721
+- Open Zeppelin ERC1155
+- 0x Project ERC20
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:62]: Same for `0x Project`

File: docs/file_explorer.md [hunk 6]
```diff
@@ -1,68 +1,108 @@
...
-- **C.** Delete a Workspace  <br>
--  **D.** Download all Workspaces This will create a .zip file with all the files of all the Workspaces.  The zip file will have a folder called **.workspaces** that will contain a folder of each workspace.  Depending on your OS, you may need to change the preferences on .workspaces folder to make it visible.  <br>
-- **E.** Upload the Workspaces backup made from the previous icon. <br>
-- **F.** Clone a repo from Git repository <br>
-- **G.** Link to documentation <br>
-- **J.** Publish the Workspace to a GIST <br>
-- **L.** Choose a Workspace 
-### File Manipulation
-The letters in bold below refer to the labels in fig. 1.
+When choosing an Open Zeppelin template, additional functionality can be added.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:65]: Same here

File: docs/file_explorer.md [hunk 7]
```diff
@@ -1,68 +1,108 @@
...
-## Workspaces & cloning a repo
-When you click the **clone** icon (**F.** in fig.1), you'll be asked for the url of the repo.  A new workspace will be created that will contain the cloned repo.  To manage the Git repo, go to the DGIT plugin.
+##### Clone
+When clicking Clone, you’ll be asked for the url of a remote repo. A new workspace will be created that will contain the cloned repo. To manage the Git repo, go to the DGIT plugin.
+##### Backup
+Backup is for downloading all the Workspaces in a .zip file. The zip file will have a folder called **.workspaces** that will contain a folder of each Workspace.  Depending on your OS, you may need to change the preferences on .workspaces folder to make it visible.  
 
-## Workspaces with a Git repo
+##### Restore
+Restore is only for uploading a .workspaces folder that was created using the Backup operation.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:80]: it is better to say `Uploading the backup zip file`

File: docs/file_explorer.md [hunk 8]
```diff
@@ -1,68 +1,108 @@
...
+
+##### Create GitHub Actions
+The Workspace operations to create **Solidity Test Workflow**,  **Moca Chai Test Workflow**, and **Slither Workflow** are for creating GitHub actions. When clicked, a .yml file in the .workflows folder of the active Workspace.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:83]: `Moca ` should be `Mocha`

Comment (Aniket-Engg) [reply docs/file_explorer.md:83]: a .yml file `is created`

File: docs/file_explorer.md [hunk 9]
```diff
@@ -1,68 +1,108 @@
...
+
+### Workspaces initialized with Git
 Workspaces with an associated Git will have the Git icon next to them in the Workspaces select box.
 
 ![](images/a-fe-select-git.png)
 
+When creating a new Workspace, there is a checkbox in the modal for choosing the template that will initialize the Workspace for Git.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:90]: `modal for choosing the template` seems confusing

File: docs/file_explorer.md [hunk 10]
```diff
@@ -1,68 +1,108 @@
...
+
+Working with Files
+-------------------
+When a file is clicked on it will appear in the Editor.
+
+Files can be manipulated by right clicking on them ([see below](#right-click-on-a-file-or-folder)) and also by using the icons under the Workspace select box.
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:96]: `Files can be manipulated by right clicking on them` seems confusing to me

File: docs/file_explorer.md [hunk 11]
```diff
@@ -1,68 +1,108 @@
 File Explorer
 =============
+The File Explorer is for managing Workspaces and files.  There is also a context menu that pops up when you right click on a file or folder.
```
[the remark this answers is not in the document]
Comment (vermouth22) [reply docs/file_explorer.md:3]: Re: "Workspace" vs. "workspace" ... If you follow a standard wherein it's like a "Project" either here or in other apps, it should be capitalized "Workspace" in all UI labels, etc. but not capitalized i.e. "workspace" when talking about it in documentation body copy. I have been making this mistake in our Medium articles unfortunately.

File: docs/file_explorer.md [hunk 12]
```diff
@@ -1,68 +1,108 @@
...
 File Storage
 ------------
 By default, Remix IDE stores files in **IndexedDB**.  
 
-Coding in Remix IDE Online is different from writing in a Google doc. Yes, both are written in a browser but a Google doc saves your work to Google’s servers, and Remix—out of the box—only saves your code to your browser’s storage. So tread carefully, browser storage is not permanent!
+Coding in Remix IDE Online is different from writing in a Google doc. 
+- A Google doc saves your work to your account on Google’s servers.
+- Remix has no user accounts.  By default, files are ONLY saved locally in the browser’s storage. So tread carefully, browser storage is not permanent!
+
+See below for using other storage locations - notably **Remixd** to connect Remix to a specific folder on your hard drive.
```
[the remark this answers is not in the document]
Comment (vermouth22) [reply docs/file_explorer.md:21]: In general, some sentences have two spaces after a period. Others have one space. It should be standardized to only one space after each period.

File: docs/file_explorer.md (hunk 10 above)
Comment (ryestew) [reply docs/file_explorer.md:96]: updated

File: docs/file_explorer.md [hunk 13]
```diff
@@ -1,68 +1,116 @@
...
-If you want to store files on your computer's filesystem, use **[Remixd](remixd.html)** or use the **[desktop version of Remix IDE](https://github.com/ethereum/remix-desktop/releases/)**. Remixd enables you to have access to a selected folder on your hard drive. Remix Desktop is a version of Remix IDE in an Electron app. 
+### Connecting Remix to remote Git repos
+**If you are not using Remixd, it is highly recommended that you save to a remote repo.** 
+(Browsers do crash causing localstorage & indexedDB to be corrupted)
+
+Remix IDE can be connected to remote Git repos hosted in GitHub (or similar service) or in IPFS.  
+Most of the Git operations are done in the **DGIT** pluin. (DGIT stands for Decentralized GIT).
+
+Branch management is also available at the bottom of the File Explorer when the Workspace is Git initialized.
+(see this page for the Git workflow).
```
Comment (Aniket-Engg) [inline docs/file_explorer.md:46]: Are you considering to add a link on `this page`

File: docs/file_explorer.md [hunk 14]
```diff
@@ -85,28 +133,27 @@ Take the token and paste it in Remix's **Settings** module in the **Github Acces
...
 -------------------------------
-
+### Right-Click on a folder
 ![](images/a-fe-rtclick-file.png)
 
-Right-clicking on a file or folder will bring a context menu — where you can create a folder or file within the same folder or to delete, rename, or publish the file or folder. 
+Right-clicking on a folder will bring a popup menu for operations you can do on that folder. 
 
-The functionality of the context menu also works with RemixD (which gives you have access to a folder on your hard drive).  
+The right-click popup menu also works with RemixD (which gives you have access to a folder on your hard drive).
```
[the remark this answers is not in the document]
Comment (Aniket-Engg) [reply docs/file_explorer.md:143]: `gives you have access` should be updated to `gives you access`

File: docs/file_explorer.md [hunk 15]
```diff
@@ -85,28 +133,27 @@ Take the token and paste it in Remix's **Settings** module in the **Github Acces
...
+### Right-Click on a folder
 ![](images/a-fe-rtclick-file.png)
 
-Right-clicking on a file or folder will bring a context menu — where you can create a folder or file within the same folder or to delete, rename, or publish the file or folder. 
+Right-clicking on a folder will bring a popup menu for operations you can do on that folder. 
 
-The functionality of the context menu also works with RemixD (which gives you have access to a folder on your hard drive).  
+The right-click popup menu also works with RemixD (which gives you have access to a folder on your hard drive).  
 
 **Note:** When working with RemixD, and when adding files to the shared folder from your computer (and not from Remix), you'll need to open and close the containing folder or switch in and out of **localhost** workspace to refresh the view.
```
[the remark this answers is not in the document]
Comment (Aniket-Engg) [reply docs/file_explorer.md:145]: Also, I think this thing is fixed now. It syncs the changes immediately. please check

File: docs/file_explorer.md (hunk 1 above)
Comment (ryestew) [reply docs/file_explorer.md:43]: We have had users ask what DGIT means, so its worth it to leave it.

File: docs/file_explorer.md (hunk 13 above)
Comment (ryestew) [reply docs/file_explorer.md:46]: Updated - but I will be adding a separate page about Git in Remix.

**Review by Aniket-Engg — APPROVED** [review]

**Pull request merged**